test/src/test/java/org/corfudb/integration/ServerRestartTest.java

```diff
@@ -132,6 +132,37 @@ public void testSequencerRecoveryFailoverServerAndClient() throws Exception {
     }
 
     /**
+     * Test recovery with a failed over server and a new client.
+     *
+     * @throws Exception
+     */
+    @Test
+    public void testDoubleFailoverRecovery() throws Exception {
```

Comment:
Great test. I think zeeshan can add more variations to this. Including writing and reading more data.